Resizetizer.NT/Resizetizer.NT.targets

```diff
@@ -159,8 +201,110 @@
 			Overwrite="true"
 			WriteOnlyWhenDifferent="true" />
 
+		<WriteLinesToFile
+			File="$(_SharedFontInputsFile)"
+			Lines="@(SharedFont->'File=%(Identity);Link=%(Link);Alias=%(Alias)')"
+			Overwrite="true"
+			WriteOnlyWhenDifferent="true" />
+
 	</Target>
 
+	<Target Name="ProcessSharedFonts"
+		Inputs="@(SharedFont);$(_SharedFontInputsFile)"
+		Outputs="$(_SharedFontStampFile)"
+		AfterTargets="$(ProcessSharedFontsAfterTargets)"
+		BeforeTargets="$(ProcessSharedFontsBeforeTargets)"
+		DependsOnTargets="$(ProcessSharedFontsDependsOnTargets)">
+
+		<DetectInvalidResourceOutputFilenamesTask
+			Items="@(SharedFont)"
+			ErrorMessage="One or more invalid file names were detected.  File names must be lowercase, start and end with a letter character, and contain only alphanumeric characters or underscores: &#13;&#10;">
```

Comment:
Maybe the newline should be in the task? 

Also, the two messages look the same? Maybe it is fine to have the default as this in the task and we can override in the targets if need be?

Reply:
Was thinking more for translations, keeping the messages in the .targets...  I refactored into a shared property at least now.